openapi/index.yaml

```diff
@@ -545,6 +545,8 @@ definitions:
         $ref: '#/definitions/IsTestProfile'
       last_app_version:
         $ref: '#/definitions/AppVersion'
+      push_notifications_content_type:
+        $ref: '#/definitions/Profile/properties/push_notifications_content_type'
```

Comment:
Add into `index.yaml.template` the type definition for `PushNotificationContentType`

Reply:
done in https://github.com/pagopa/io-functions-app/pull/260/commits/5a7b04ee292edaacf67e1cb1992bfab4dd723f45